tests/test_diff_text.py

```diff
@@ -1465,3 +1492,10 @@ def test_diff_when_hash_fails(self, mock_DeepHash, mock_logger):
         t2 = {"blah": {4}, 2: 1337}
         DeepDiff(t1, t2, ignore_order=True)
         assert mock_logger.warning.called
+
+    def test_diff_netaddr_eui(self):
+        a = EUI('11:11:11:11:11:11')
```

Comment:
@brianmaissy Netaddr seems like a cool library. Didn't know about it. But I wouldn't add it as a dependency of DeepDiff if it is just for testing purposes. Were you dealing with EUI objects that ran into the bug? Can you please recreate enough of the EUI for testing purposes in this test file without having to install the whole package? Thanks!

Reply:
removed

Reply:
The other tests cover the underlying bugs. The netaddr test was just for good measure.